takealot.user.js

```diff
@@ -0,0 +1,563 @@
+// ==UserScript==
+// @name           Import Takealot releases to MusicBrainz
+// @description    Add a button to import Takealot releases to MusicBrainz
+// @version        2016.03.09.2
+// @namespace      http://lenjoubertblog.blogspot.com
+// @include        http*://www.takealot.com/*
+// @require        https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/mbimport.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/logger.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/mblinks.js
+// @require        https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/lib/mbimportstyle.js
+// @icon           https://raw.githubusercontent.com/murdos/musicbrainz-userscripts/master/assets/images/Musicbrainz_import_logo.png
+// @grant          none
+// ==/UserScript==
+
+
+// prevent JQuery conflicts, see http://wiki.greasespot.net/@grant
+this.$ = this.jQuery = jQuery.noConflict(true);
+
+if (!unsafeWindow) unsafeWindow = window;
+
+var DEBUG = true;
```

Comment:
Please set `DEBUG` to `false`, as final user doesn't want it enabled by default.